toolkit/committee-selection/authority-selection-inherents/src/filter_invalid_candidates.rs

```diff
@@ -256,11 +292,10 @@ pub fn validate_registration_data(
 	)?;
 	verify_tx_inputs(registration_data)?;
 
-	// TODO - Stake Validation: https://input-output.atlassian.net/browse/ETCM-4082
-
 	Ok((sidechain_pub_key, (aura_pub_key, grandpa_pub_key)))
 }
 
+/// Validates stake delegation. Stake must be stable and positive.
```

Comment:
```suggestion
/// Validates stake delegation. Stake must be known and positive.
```
Because there can be said nothing about stability in this context.